src/App.js

```diff
@@ -1,23 +1,56 @@
-import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
+import contactsArr from './contacts.json';
 
 function App() {
+
+  const arrOfFive = contactsArr.slice(0, 5).map((elm) => {
+    return elm
+  })
```

Comment:
Slice() method returns a new array containing the extracted elements. So mapping is not needed here